.tests/cypress/integration/mini-cart/remove-from-cart-configurable.js

```diff
@@ -0,0 +1,97 @@
+function pickRandomitem (item) {
+  return item[Math.floor(Math.random() * item.length)]
+}
+
+describe('Delate Configurable product', function () {
+  before(() => {
+    // Hide premissions popup
+    cy.setCookie('permission-cookies', 'true')
+    cy.setCookie('permission-profiling', 'true')
+    cy.setCookie('mage-cache-sessid', 'true')
+    // Keep cookies beween tests
+    Cypress.Cookies.defaults({
+      whitelist: [
+        'frontend',
+        'X-Magento-Vary',
+        'permission-profiling',
+        'PHPSESSID',
+        'form_key'
+      ]
+    })
+  })
+
+  it('Visits product', () => {
+    cy.visit('/configurable-product')
+    cy.get('.breadcrumbs__list').should('be.visible')
+  })
+
+  it('Check visiblility of content', () => {
+    cy.get('.product-view__main-details').should('be.visible')
+    cy.get('[data-ui-id=page-title-wrapper]').should('be.visible')
+    cy.get('.product-view__information').should('be.visible')
+    cy.get('.product-view__sku').should('be.visible')
+    cy.get('#product-price-2048').should('be.visible')
+    cy.get('.quantity-update').should('be.visible')
+    cy.get('#product-addtocart-button').should('be.visible')
+    cy.get('.product-view__extra-actions').should('be.visible')
+    cy.get('[data-testid=product-tab__title]').contains('Details').should('be.visible')
+    cy.get('[data-testid=product-tab__title]').contains('Reviews').should('be.visible')
+  })
+
+  it('Test reviews tab', () => {
+    cy.get('[data-testid=product-tab__title]').contains('Reviews').click()
+    cy.get('.product-review__title').contains('Customers reviews')
+    cy.get('#review-form').should('be.visible')
+    cy.get('#reviewAddButton').should('be.visible')
+  })
+
+  it('Checks configurable options', () => {
+    cy.get('.swatch-opt').should('be.visible').find('.size')
+    cy.get('.size').find('.swatch__option')
+      .then((elements) => {
+        cy.log(elements)
+        pickRandomitem(elements).click()
+      })
```

Comment:
```suggestion
    pickRandomitem(cy.get('.size .swatch__option')).click()
```

Reply:
Suggestion solution resulted to error `TypeError: Cannot read property 'click' of undefined` which I think it's as of result of `pickRandomitem()` not being a cypress command, thus we cannot chain `.click()` action to it. So I created it as a cypress custom command, but this time the test gets stuck when the command is called.